python/jupyter_notebook/cipherstash_encrypt_eql.sql

```diff
@@ -0,0 +1,819 @@
+DROP CAST IF EXISTS (text AS ore_64_8_v1_term);
```

Comment:
The eql sql has been moved to this repo, so you can point to the [release folder](https://github.com/cipherstash/encrypt-query-language/tree/main/release) instead of having to copy this over.